app/api/main.py

```diff
@@ -1,6 +1,6 @@
 from fastapi import APIRouter
 
-from app.api.routes import chat
+from app.api.routes import chat 
```

Comment:
`nit`
https://peps.python.org/pep-0008/#other-recommendations
> Avoid trailing whitespace anywhere. Because it’s usually invisible, it can be confusing: e.g. a backslash followed by a space and a newline does not count as a line continuation marker. Some editors don’t preserve it and many projects (like CPython itself) have pre-commit hooks that reject it.

Reply:
Thank you for your review! Yes, in fact, I forgot to run ruff and black on this code. 
I will read the code styling guideline, pay attention while coding, and check my codes using tools.

Reply:
I think being aware of all the rules in PEP8 is very hard, especially for beginners 😅 
I would recommend to use tools like `flake8`, `ruff` (not working? 🤔 ), or `black` instead of reading the documentation.